e2e/cypress/integration/ui/settings/02_team_mgmt.spec.ts

```diff
@@ -41,11 +63,251 @@ describe('team management', () => {
+    const STORE_OPTIONS_KEY = 'projectsFilter.options';
+    const dropdownNameUntilEllipsisLen = 25;
+
+    context('when only the unassigned project is selected', () => {
+      before(() => {
+        localStorage.setItem(STORE_OPTIONS_KEY, JSON.stringify([
+          {label: unassigned, value: unassigned, checked: true},
+          {label: project1Name, value: project1ID, checked: false},
+          {label: project2Name, value: project2ID, checked: false},
+          {label: unusedProject, value: unusedProject, checked: false}
+        ]));
+
+        // relog to apply localstorage state
+        cy.adminLogin('/settings/teams').then(() => {
+          const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+          adminToken = admin.id_token;
+        });
+      });
+
+      it('can create a team with no projects (unassigned) ' +
+      'and cannot access projects dropdown', () => {
+        cy.get('app-team-management chef-toolbar chef-button').contains('Create Team').click();
+        cy.get('app-team-management chef-modal').should('exist');
+        cy.get('[data-cy=create-name]').type(teamName);
+        cy.get('[data-cy=id-label]').contains(generatedTeamID);
+
+        // initial state of dropdown
+        cy.get('app-projects-dropdown #projects-selected').contains(unassigned);
+
+        // TODO (tc) why doesn't this work?
+        // cy.get('app-projects-dropdown .dropdown-button').should('have.attr', 'disabled', true);
+
+        // save team
+        cy.get('[data-cy=save-button]').click();
+        cy.get('app-team-management chef-modal').should('not.be.visible');
+        cy.get('chef-notification.info').should('be.visible');
+        cy.contains(teamName).should('exist');
+        cy.contains(generatedTeamID).should('exist');
+        cy.get('[data-cy=team-details-projects]').contains(unassigned);
+        cy.go('back');
       });
- });
+    });
+
+    context('when there are multiple custom projects selected in the ' +
+      'filter (including the unassinged project)', () => {
+      before(() => {
+        cy.request({
+          auth: { bearer: adminToken },
+          method: 'POST',
+          url: '/apis/iam/v2beta/projects',
+          body: {
+            id: project1ID,
+            name: project1Name
+          }
+        });
+
+        cy.request({
+          auth: { bearer: adminToken },
+          method: 'POST',
+          url: '/apis/iam/v2beta/projects',
+          body: {
+            id: project2ID,
+            name: project2Name
+          }
+        });
+
+        localStorage.setItem(STORE_OPTIONS_KEY, JSON.stringify([
+          {label: unassigned, value: unassigned, checked: true},
+          {label: project1Name, value: project1ID, checked: true},
+          {label: project2Name, value: project2ID, checked: true},
+          {label: unusedProject, value: unusedProject, checked: false}
+        ]));
+
+        // relog to apply localstorage state
```

Comment:
why do we have to relog here? can you not set the state after logging in?

Reply:
We just need to update localstorage before it loads the page, so it gets applied. If we update after the teams page loads it'll be too late.